.github/workflows/canary-integration-test.yml

```diff
@@ -694,7 +694,7 @@ jobs:
       if: failure() && contains(github.event.pull_request.labels.*.name, 'debug-encryption-pvc-db-wal')
       uses: mxschmitt/action-tmate@v3
 
-  encryption-pvc-kms-vault-token-auth:
+  encryption-kms-vault-token-auth:
```

Comment:
Why removing `pvc-` from this? I see `pvc` was added to `validate pvc vault` later on, and this still uses `test-cluster-on-pvc-encrypted.yaml`.